.azure-devops/test/jobs/steps/parallel.yml

```diff
@@ -0,0 +1,66 @@
+steps:
+    - script: |
```

Comment:
and for those dependencies, you can check our python script, we moved all those install dependencies into a python program so it is easier to manage in the future.